symfit/core/support.py

```diff
@@ -87,9 +112,11 @@ def f(x, p):
 
     return f
 
+@deprecated('symbols(names, cls=Variable)')
 def variables(names, **kwargs):
```

Comment:
I disagree with the deprecation of variables and parameters. These are convenient helper functions